### ISSUE TITLE
"Cancel" button

### ISSUE THREAD

**Issue by maneesha** [swcarpentry/amy#881]
Any place there is a "Submit" button I'd also like to see a "cancel" button so that any changes accidentally made while in editing mode are not saved.

**Comment by narayanaditya95** [comment]
A simple force reload of the page will refetch the content of form fields from the server.
It can be achieved through `Ctrl + Shift + R` or alternatively we can have a button which calls `location.reload(True)`.

**Comment by maneesha** [comment]
@narayanaditya95 yes, that works but I'd rather have an explicit "cancel" button than to rely on browser activity like back or refresh (which is what I do now).

**Comment by narayanaditya95** [comment]
@chrismedrela 
I noticed that `BootstrapHelper` now adds support for `add_delete_button`. However, the delete button does not generate any functionality except for showing an alert.
Am I missing something?

**Comment by chrismedrela** [comment]
@narayanaditya95 please have a look at [TrainingProgressForm](https://github.com/swcarpentry/amy/blob/beafe92190e16f8d9eab2f0e6fe7677098d27d8e/workshops/forms.py#L1218). In the admin dashboard, you can click "more > trainees", then add progress record, then click it to edit it, and a working "delete" button is displayed.

Could you share the code that does not generate any functionality except for showing an alert?
